src/css/components/_progress.scss

```diff
@@ -0,0 +1,52 @@
+.video-js .vjs-progress-control {
+  @include flex(auto);
+  @include display-flex(center);
+
+  @include transition(all 0.4s);
+}
+
+/* Box containing play and load progresses. Also acts as seek scrubber. */
+.video-js .vjs-progress-holder {
+  @include flex(auto);
+  height: 0.3em;
+}
+
+/* Progress Bars */
+.video-js .vjs-progress-holder .vjs-play-progress,
+.video-js .vjs-progress-holder .vjs-load-progress,
+.video-js .vjs-progress-holder .vjs-load-progress div {
+  position: absolute;
+  display: block;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  /* updated by javascript during playback */
+  width: 0;
+  /* Needed for IE6 *///
+  left: 0;
+  top: 0;
+}
+
+.video-js .vjs-play-progress {
+  background-color: $primary-text;
+}
+.video-js .vjs-load-progress {
+  background: rgb(100, 100, 100) /* IE8- Fallback */;
+  background: rgba(255, 255, 255, 0.2);
+}
+
+/* there are child elements of the load progress bar that represent the
+specific time ranges that have been buffered */
+.video-js .vjs-load-progress div {
+  background: rgba($secondary-bg, 0.1);
+}
+
+.video-js .vjs-slider-handle.vjs-seek-handle {
```

Comment:
Could we reorder the includes so this doesn't need the double class? It doesn't look like slider-handle currently sets these values anyway, but it probably should so you can use a default slider.